### PR TITLE
IBX-6636: ezSelection default label 'ALL' is incorrect for NOT multiple choice setting

### PR DESCRIPTION
**Pull request by vidarl**
| :ticket: Issue | IBX-6636 |
|----------------|-----------|

The text `ALL` in the selection placeholder didn't make any sense. Changed it to `None`

image

#### For QA:
- Create ContentType with ezselection ( multiselection *NOT* enabled ) with choices
  - BMW
  - Audi
  - Ford
- Create content with new ContentType and check the placeholder

### CODE REVIEW

**Review by adamwojs — APPROVED** [review]
It definitely make sense for me, but let's double check with @juskora and @NataliaBecla .

**Comment by KatarzynaSobczak** [comment]
I would stick to the rule in our [Design System](https://www.figma.com/design/4sbNQyNLhFFwxWDH89Rf9X/%F0%9F%9F%A2-%5BIbexa%5D-Design-System?m=auto&node-id=3080-86940&t=uaBPD3BUCtmyCYNr-1) but the logic is the same as Radek wrote in the comment of task [IBX-6021](https://issues.ibexa.co/browse/IBX-6021) - "The usage of "all" as default value in selection/drop-down should depend on the context. General rule: The default label in the selection field should not be "All" if this is not a multiple-choice selection. In practice:
- dropdown used as a filter should contain ALL, because before selecting a specific value it means that the table displays all possible values (like in JIRA filters) e.g. Status in Orders, Catalogs
- dropdown used in the selection attribute, currency selector, VAT category should not contain ALL, because you need to select only one value. ALL is not logical in this case."

Long story short:
- adding 'all' in the dropdown should depend on the context
- with a single-selection there's no point for 'all' 
- I would stick to the proposed solution of not displaying 'all'

**Comment by juskora** [comment]
I agree with Kasia, and what's more, for an empty state there should be "Choose an option" not "None" as suggested.

**Comment by KatarzynaSobczak** [comment]
@juskora great point!

**Comment by vidarl** [comment]
@juskora, @KatarzynaSobczak 

Changing it to "Choose an option" has one disadvantage IMO.

Let's say user as already select something ( in this case "Audi"). Now he wants to unselect it...
User then needs to click on "Choose an option" in order to unselect it:

image

Would it be an idea to label the placeholder "None" if the field is not required, and label it "Choose an option" only of field is required?

**Comment by vidarl** [comment]
ping @juskora, @KatarzynaSobczak

**Comment by IdaDraM** [comment]
We discussed issue with naming and behaviour for dropdowns and recommend:
 - For multichoice: Choose an option or All
 - For single select: Choose an option
 Each dropdown should have X clear icon, so user can change state to enabled (not filled). 
It will be added to Design System.
 Designs: https://www.figma.com/design/C84ggnTTrPZa2X0dj23IQ7/Bugs-reported-in-v4.6?node-id=1687-126946&t=ySKCIpUTOGNmIAYE-1
